client/my-sites/plans/current-plan/jetpack-product-install/index.tsx

```diff
@@ -278,10 +278,14 @@ export class JetpackProductInstall extends Component< Props, State > {
 
 		if ( hasErrorInstalling && ! this.tracksEventSent ) {
 			this.tracksEventSent = true;
+			const { status } = this.props;
+
 			this.props.recordTracksEvent( 'calypso_plans_autoconfig_error', {
 				checklist_name: 'jetpack',
 				error: 'installation_error',
 				location: 'JetpackChecklist',
+				status_akismet: status ? status.akismet_status : '(unknown)',
```

Comment:
Curious, if you don't mind me asking: Is `(unknown)` a convention of sorts, or does it have a special meaning at the place it is processed?

Reply:
Nothing special, just something I hope conveys we don't have data 🙂

Reply:
Ah thanks for responding. Makes sense. Would `null | undefined` work as well in that case? I sense it might even be easier to type that way later too. 

e.g. assuming these don't have to be strings, then this seems natural (flow):

```
type properties = {|
    checklist_name: string,
    ...,
    status_akismet: ?AkismetStatus, // else ?string
|}
```

Sorry if wasting time here. :D

Reply:
Tried to check docs if we have some standard around undefined/null/unknown and couldn't find anything mentioned. @AnnaMag might know?

I'd say good to deploy as-is and we can follow up with correct one in separate PR if we find a better value.

Reply:
> I'd say good to deploy as-is and we can follow up with correct one in separate PR if we find a better value.

Hmmm I am not familiar with "tracks", but having flags like this going undecided and modified down the line can be a source of headaches. Not saying that "unknown" is a bad name - just hoping we are fairly decisive before merging (since it's been brought up).

I think these types of things need to be decided/finalised as upfront as possible. I can foresee queries of the sort "all entries with a null status as of xyz date" - while some "null" ones were recorded as "(unknown)" for a while but nobody bothered to migrate. Could this be a potential scenario? How exactly does a migration like this happen?

Reply:
This is good feedback. I don't think there's anything standardized largely because each event is unique. There are a set of global properties all events should have, but each distinct event may have unique properties. The important bit is that individual events are mostly consistent, but I don't think it matters too much whether the value for a particular property unique to an individual event is `null` or a textual representation that attempts to be more descriptive.

I typically assume the value `null` is an oversight, something we didn't expect to find and didn't cover. By supplying a descriptive string value `(unknown)`, we recognize that that absence of a value is a valid and expected state.

The reality is that it probably doesn't matter too much one way or the other 🙂 When working with tracks events and event-specific properties, you end up looking at the possible values based on what you find in tracks anyways.

Reply:
> I typically assume the value null is an oversight, something we didn't expect to find and didn't cover. By supplying a descriptive string value (unknown), we recognize that that absence of a value is a valid and expected state.

I like this. 👍  We see a lot of `null` in Tracks just because the prop was never added to an event somewhere.